CHANGELOG.md

```diff
@@ -3,6 +3,48 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.16.0
```

Comment:
I thought it would be a patch. The quick filtering logic modification is kind of a bug fix

Reply:
I have to agree on that, even though there are certain changes in "API", but it's not related to `DataGrid`
For example: the `pickers` defaulting to `desktop` mode in tests, but those have their own `beta` version. 👍